OpenRA.Mods.Common/Traits/Conditions/GrantConditionOnBotOwner.cs

```diff
@@ -42,10 +42,6 @@ public GrantConditionOnBotOwner(ActorInitializer init, GrantConditionOnBotOwnerI
 
 		void INotifyCreated.Created(Actor self)
 		{
-			// Special case handling is required for the Player actor.
-			// Created is called before Player.IsBot is set, so we
-			// must use a different method to enable this trait if
-			// it's defined on the PlayerActor.
```

Comment:
Shouldn't this comment stay?

Reply:
Ah derp. I wanted to fix the workaround here too but didn't, thanks.